core/io/autoclose_spec.rb

```diff
@@ -0,0 +1,49 @@
+require_relative '../../spec_helper'
+require_relative 'fixtures/classes'
+
+describe "IO#autoclose" do
+  before :each do
+    @io = IOSpecs.io_fixture "lines.txt"
+  end
+
+  after :each do
+    @io.autoclose = true unless @io.closed?
+    @io.close unless @io.closed?
+  end
+
+  it "can be set to true" do
+    @io.autoclose = true
+    @io.should.autoclose?
+  end
+
+  it "can be set to false" do
+    @io.autoclose = false
+    @io.should_not.autoclose?
+  end
+
+  it "can be set to any truthy value" do
+    @io.autoclose = 42
+    @io.should.autoclose?
```

Comment:
**suggestion:** It makes sense to set `false` and then to assign truthy value - this way we ensure assigning truthy value isn't ignored and affects an io object.

Also it makes sense to assign different truthy values (`Object.new`, `[]`, `"abc"`, etc) in the same test case as an illustration that there is no special logic for `Integer` type (and Integer value `42`)

Reply:
I added `Object.new` as well. That, combined with the test description of "any truthy value" should be clear enough.